modules/core/src/shaderlib/project/project.glsl.ts

```diff
@@ -62,6 +62,11 @@ const float EARTH_RADIUS = 6370972.0; // meters
 const float GLOBE_RADIUS = 256.0;
 
 // returns an adjustment factor for uCommonUnitsPerMeter
+float project_size_at_latitude(float lat) {
+  float y = clamp(lat, -89.9, 89.9);
+  return 1.0 / cos(radians(y));
```

Comment:
Due to the symmetry you could do `1.0 / max(cos(radians(lat)), 0.001745328365898)`. I have a memory that `max` is more performant than `clamp` but I can't find the source

Reply:
I'll make a separate PR for this. Keeping this one clean for a patch.